frontend/src/models/api/Property.ts

```diff
@@ -0,0 +1,42 @@
+import { Api_Address } from './Address';
+import { Api_ConcurrentVersion } from './ConcurrentVersion';
+import Api_TypeCode from './TypeCode';
+
+export interface Api_Property extends Api_ConcurrentVersion {
```

Comment:
I could not find a new PropertyModel on the Concepts namespace. Is this mapping to the old one or one of the feature-specific ones? If so, I believe it would be better if we have a property model on the front end that is specific to that one until we get the general one on the backend

Reply:
PropertyModel was already existing on it's own Area and other controllers/logic depend on it so I felt that refactoring it to a common "concept" was not part of this PR. Can be addresses later I think

Reply:
Yeah I agree. This model, however, is a new one that should be common (Api_Property). If we are using the old model on the backend it should map to the old model on the front end.